source/components/panel/index.html

```diff
@@ -0,0 +1,75 @@
+---
+title: Panel
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 class="hxSectionTitle" id="summary">Summary</h2>
+    <p>
+      A panel is a utility for sectioning content vertically into a head,
+      body, and foot.
+    </p>
+    <ul>
+      <li>The head is fixed to the top of the panel.</li>
+      <li>The foot is fixed to the bottom of the panel.</li>
+      <li>The body fills the space in the middle.</li>
+    </ul>
+  </section>
+  <section id="vue-panelDemo">
+    <h2 class="hxSectionTitle" id="demo">Demo</h2>
+    <div class="hxRow">
+      <div class="hxCol hxSpan-12-xs hxSpan-8-md">
+        <div class="demo panel-demo">
+          <hx-panel>
+            <hx-panelhead v-if="hasHead">Panel Head</hx-panelhead>
+            <hx-panelbody>
+              <p>Panel Body</p>
+              <p>
+                Try to decide the branch off team, maybe it will help spike the MVP
+                interfaces. If we practice between the tasks, we can get the technical
+                application like the VOC branching. The best way to document acceptance
+                impediment is to periodically build their meeting. It was discovered
+                that by efficiently releasing the MVP products, we can practice the
+                lean XP development team in the minor board.
+              </p>
+            </hx-panelbody>
+            <hx-panelfoot v-if="hasFoot">Panel Foot</hx-panelfoot>
+          </hx-panel>
+        </div>
+        <small class="hxItalic">
```

Comment:
Double-check within the typography file to see if caption styling can be used instead.